src/pool/srv_layout.c

```diff
@@ -45,6 +45,7 @@ RDB_STRING_KEY(ds_pool_prop_, self_heal);
 RDB_STRING_KEY(ds_pool_prop_, reclaim);
 RDB_STRING_KEY(ds_pool_prop_, owner);
 RDB_STRING_KEY(ds_pool_prop_, owner_group);
+RDB_STRING_KEY(ds_pool_prop_, accept_connections);
```

Comment:
[Optional] Could be named shorter, like "connectable".

Reply:
addressed in most recent patch

Reply:
Thanks.